ONSdigital/census-rm-case-processor
# Add batch ID to uac qid links

# Motivation and Context
We need to be able to identify batches of unaddressed uac qid pairs, the easiest solution for this was to add in a UUID batch_id column which is set off the unaddressed request messages.

# What has changed
- Add batch ID to uac qid links
- Add batch ID to unaddressed request message
- Set batch ID in unaddressed request message reciever
- Update tests

# How to test?
Build and run this branch, then run through generating an unaddressed batch with the linked qid-batch-runner branch. You should be able to run multiple batches of unaddressed pairs without failing on generating the print files.

# Links
https://github.com/ONSdigital/census-rm-qid-batch-runner/pull/3
https://github.com/ONSdigital/census-rm-case-api/pull/6
https://github.com/ONSdigital/census-rm-ops/pull/10
https://trello.com/c/sOyLOxiP/783-add-batch-identifier-to-unaddressed-qid-uac-pairs-5
